operator/deploy/olm-catalog/ibm-spectrum-scale-csi-operator/manifests/ibm-spectrum-scale-csi-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,467 @@
+---
```

Comment:
Some of the content of this file are not correct.
Rather copy https://raw.githubusercontent.com/IBM/ibm-spectrum-scale-csi/master/operator/deploy/olm-catalog/ibm-spectrum-scale-csi-operator/manifests/ibm-spectrum-scale-csi-operator.v2.1.0.clusterserviceversion.yaml as ibm-spectrum-scale-csi-operator.clusterserviceversion.yaml  and make necessary changes